fetch/origin/resources/redirect-and-stash.py

```diff
@@ -20,6 +20,13 @@ def main(request, response):
 
     request.server.stash.put(key, origin_list)
 
+    if "referrerPolicy" in request.GET:
+        response.status = 200
+        response.headers.set("content-Type", "text/html")
+        response.headers.set("Referrer-Policy",
+                             request.GET.first("referrerPolicy"))
+        return
```

Comment:
This block could be much higher up, right? It might even be good to have this in a separate file to indicate it doesn't really affect all the stash logic.

(I guess with this setup you're implicitly testing the GET scenario, but that really seems too subtle. I'd much rather we test that explicitly as a distinct set of tests from the POST tests or as some kind of permutation upon them.)